sympy/geometry/point3d.py

```diff
@@ -508,7 +506,7 @@ def translate(self, x=0, y=0, z=0):
 
     def transform(self, matrix):
         """Return the point after applying the transformation described
-        by the 3x3 Matrix, ``matrix``.
+        by the 4x4 Matrix, ``matrix``.
```

Comment:
This seems to be changing the functionality of a public method. Previously just the rotation matrix was returned, I assume? And now a transformation matrix is?

If this a necessary change it needs to be deprecated.

https://github.com/sympy/sympy/wiki/Deprecating-policy

Another option would be to have a new method that returns a transformed point.

Can you explain this change.